apps/api/src/domain/models/photograph/queries/photograph.view-model.ts

```diff
@@ -53,11 +54,18 @@ export class PhotographViewModel implements HasAggregateId, DetailScopedCommandW
     public id: string;
 
     @ApiProperty()
-    @NonEmptyString({
-        label: 'photographer',
-        description: 'the full name of the photographer responsible for this photograph',
+    @PositiveInteger({
+        label: 'Image Height PX',
+        description: 'Height of the image in pixels',
     })
-    public photographer: string;
+    public heightPx: number;
+
+    @ApiProperty()
```

Comment:
Soon we won't need the `ApiProperty` decorator. Instead, we're going to use reflection to get the COSCRAD data schema and a util that maps this to the Open API schema standard.